ui/desktop/app/templates/scopes.hbs

```diff
@@ -3,4 +3,51 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-{{outlet}}
\ No newline at end of file
+{{outlet}}
+{{#unless this.isPaginationSupported}}
```

Comment:
Comment for sake of visibility:

Already mentioned to @cameronperera in slack and working on it, this comment is for visibility purposes.

In linux, the `<modal>` overlay, does not allow the user to click the close app button on the header, top - right corner (attached screenshot).
![Screenshot from 2024-01-11 16-15-07](https://github.com/hashicorp/boundary-ui/assets/9775006/1bea0cda-3e77-47bd-b583-44071e48d3fc)

The user can always go to the desktop dock, right click on Boundary app icon and close it (attached screenshot)
![Screenshot from 2024-01-11 16-21-24](https://github.com/hashicorp/boundary-ui/assets/9775006/bbc896eb-1aa9-4481-ae96-53dfaab01242)

Not a big deal. I suggest since we are tight on delivery time, perhaps we can leave this for after release work?

Reply:
Thanks for noticing this. I will look into this but like you said, probably not a blocker for release.